frontend/pulse3d/src/components/layouts/DashboardLayout.js

```diff
@@ -74,7 +74,13 @@ export default function DashboardLayout({ children }) {
       value={{ uploads, setUploads, setFetchUploads, pulse3dVersions, metaPulse3dVersions }}
     >
       <Container>
-        <ControlPanel />
+        <ControlPanel
+          clearTimers={() => {
+            if (clearTimers) {
```

Comment:
I think it's better to leave the check in ControlPanel and remove it here since it's actually getting called in `ControlPanel`. If another component were to instantiate a `ControlPanel` it would also need to add this check, vs. if we just put the check in `ControlPanel` then it will only need to be done once no matter how many other components use `ControlPanel`